deployments/puppet/metadata.json

```diff
@@ -9,6 +9,14 @@
     {
       "name": "puppetlabs/apt",
       "version_requirement": ">= 4.1.0 <= 7.0.0"
+    },
+    {
+      "name": "puppet/archive",
+      "version_requirement": ">= 5.5.8 < 7.0.0"
+    },
+    {
+      "name": "puppetlabs/powershell",
+      "version_requirement": ">= 4.7.0 < 7.0.0"
```

Comment:
https://forge.puppet.com/puppetlabs/powershell has 1.0.1 to 2.3.0 as available versions.

Reply:
Same as above

Reply:
corrected